src/components/ApiRefRouting/Route.js

```diff
@@ -13,12 +13,10 @@ export const Route = ({ children, originalFilePath, path }) => {
     ScrollRouterContext,
   );
   const ref = React.useRef();
-  const childRef = React.useRef();
   React.useEffect(() => {
-    childRef.current = ref.current.children[0];
-    trackElement(childRef, path);
+    trackElement(ref, path);
     return () => {
-      stopTrackingElement(childRef);
+      stopTrackingElement(ref);
```

Comment:
By tracking the first child, It ended up tracking the heading. Since the logic I wanted to use is based on watching the top and bottom of the content, that doesn't work as desired. Track the wrapper div.